### ISSUE TITLE
escape quits working to go back to command output

### ISSUE THREAD

**Issue by dannyneira** [warpdotdev/Warp#1817]
### Describe the bug

I have no idea how this happens, but sometimes escape just quits working to go back to command output.

This really needs to be fixed because the only thing I found to fix it is restarting Warp.  Why does escape only work sometimes? Should it work at all?

### To Reproduce

Select a Block, press ESC, warp fails to return to input.

### Expected behaviour

Seems like cmd+down arrow will move there and cmd+l will as well, so it should work like those.

### Operating System

MacOS

### OS Version

MacOS 12.5

### Shell Version

zsh 5.8.1

### Warp Version

v0.2022.08.31.18.11.stable_00

### Does this block you from using Warp daily?

Yes
